src/styles.scss

```diff
@@ -140,22 +140,31 @@ body {
 }
 
 .error-snackbar {
-  background: #b00020;
-  color: #fff;
+  background-color: #fce0de;
```

Comment:
use our theme variable, or create one for colours

Reply:
Added and used variables for colors